robotswerve.py

```diff
@@ -47,6 +48,10 @@ def __init__(self) -> None:
             lambda: wpimath.applyDeadband(self.driveController.getRightX(), 0.1),
             lambda: self.driveTrain.getFieldDriveRelative()
         ))
+
+        if True:
```

Comment:
We should probably expose this via the dashboard...default it to true and allow drivers to turn it off? Actually let's do this instead: initially default it false so we can get a feel for it, how it performs, etc and if we like what we're seeing we can change the default to true and eventually just remove the option completely.